ui-kit/Card/Card.styles.js

```diff
@@ -99,8 +99,15 @@ const Cover = styled.div`
   ${system}
 `;
 
-const position = ({ position }) => props => {
+const position = ({ position, smallerCard }) => props => {
```

Comment:
Maybe we can do something other than `smallerCard` as the prop name? I worry about using a prop name like that when it's really related to size. Perhaps we do `size='s'` with a new `size` prop to give us some flexibility in defining other size cards. WDYT?

Reply:
yeah I'm totally fine with that, I was just trying to add the ability to remove that extra padding so the content fits correctly in smaller cards. So whatever is easiest to understand works for me!

Reply:
Yeah, this totally makes sense, but I'm trying to think about making it a little more flexible. However, that could be a premature optimization, so your call!

Reply:
Yeah I'll change it to the `size` variable so we have flexibility in the future if needed. Good call!